lib/fluent/plugin/in_prometheus/async_wrapper.rb

```diff
@@ -38,7 +39,7 @@ def get(path)
             raise error
           end
 
-          Response.new(response.status.to_s, response.body.read, response.headers)
+          Response.new(response.status.to_s, response.read || '', response.headers)
```

Comment:
This is correct.

`response.body.read` only returns first chunk.

`response.read` will return the entire body (if the response has a body) and in addition release the underlying connection back to the connection pool for future requests.